### PR TITLE
Removed # noqa from docstrings

### PR DESCRIPTION
**Pull request by ebyau**
Formating Docstrings in the ivy code base to remove "# noqa"

### CODE REVIEW

File: determine_test_coverage.py [hunk 1]
```diff
@@ -16,7 +16,9 @@
 run_iter = int(sys.argv[1])
 
 os.system(
-    "docker run -v `pwd`:/ivy -v `pwd`/.hypothesis:/.hypothesis unifyai/ivy:latest python3 -m pytest --disable-pytest-warnings ivy_tests/test_ivy --my_test_dump true > test_names"  # noqa
+    "docker run -v `pwd`:/ivy -v `pwd`/.hypothesis:/.hypothesis unifyai/ivy:latest \
+    python3 -m pytest --disable-pytest-warnings ivy_tests/test_ivy --my_test_dump \
+    true > test_names"
```
Comment (KareemMAX) [inline determine_test_coverage.py:21]: As these are system commands they shouldn't be tampered with, please revert that change

File: determine_test_coverage.py [hunk 2]
```diff
@@ -65,8 +67,10 @@
         test_name, backend = test_backend.split(",")
         print("Test:", test_backend)
         command = (
-            f'docker run -v "$(pwd)":/ivy unifyai/ivy:latest timeout 30m /bin/bash -c "coverage run --source=ivy,'  # noqa
-            f"ivy_tests -m pytest {test_name} --backend {backend} --disable-warnings > coverage_output;coverage "  # noqa
+            f'docker run -v "$(pwd)":/ivy unifyai/ivy:latest timeout 30m \
+            /bin/bash -c "coverage run --source=ivy,'
+            f"ivy_tests -m pytest {test_name} --backend {backend} --disable-warnings \
+            > coverage_output;coverage "
```
Comment (KareemMAX) [inline determine_test_coverage.py:73]: As these are system commands they shouldn't be tampered with, please revert that change

**Review by KareemMAX — CHANGES_REQUESTED** [review]
Please revert changes made in `.idea` directory. I've made some comments about places that actually are not problematic.

Regarding the other places you refactored, I guess that wasn't intended refactor. Please look into the the documentation provided by your IDE, if you looked at `ivy.data_classes.array.elementwise.logical_or()` you will find it like the (This is on my vscode):

> This function conforms to the Array API Standard <https://data-apis.org/array-api/latest/>_. This docstring is an extension of the
docstring `<https://data-apis.org/array-api/latest/API_specification/generated /signatures.elementwise_functions.logical_or.html>`_ in the standard.

There should be no space between `/generated` and `signatures`. Refer to our technical analysis in the trello card, if you have a better solution you are welcome to do it as well, as long as docstrings render correctly in both the doc-builder and vscode (or pycharm).

**Comment by ebyau** [comment]
@KareemMAX Other than the changes you requested above are the rest of the changes I made okay?

**Comment by KareemMAX** [comment]
@ebyau, I'm not really sure what changes do you mean, please reference them to me if you can

**Comment by KareemMAX** [comment]
As this PR is stale for a long time, I'll close it in the meantime.

If you wish to continue working on it in the future please feel free to reopen it

**Pull request closed without merging**